### ISSUE TITLE
send_keys to iOS input password element XCUIElementTypeSecureTextField is not working

### ISSUE THREAD

**Issue by bwomsm1** [appium/appium#15647]
## The problem

In general I am able to enter text to XCUIElementTypeTextField with no problem, my issues begin with password elements from type XCUIElementTypeSecureTextField.

This is what I have being trying (Python syntax):

- `ActionChains(self.__driver).send_keys_to_element(element, text).perform()`
- `ActionChains(self.__driver).move_to_element(element).send_keys(text).perform()`
- `ActionChains(self.__driver).send_keys(text).perform()`, I called this after performing `element.click()`
- `element.send_keys(text)`
- `self.__driver.switch_to.active_element.send_keys(text)`

## Environment

* Appium version (or git revision) that exhibits the issue: `1.18.3`
* Last Appium version that did not exhibit the issue (if applicable): Never tried with `ionic` apps
* Desktop OS/version used to run Appium: Mac OS Big Sur `11.4`
* Node.js version (unless using Appium.app|exe): `v12.9.1`
* Npm or Yarn package manager: npm
* Mobile platform/version under test: iOS `14.4`
* Real device or emulator/simulator: real device
* Appium CLI or Appium.app|exe: cli

## Details

provided in the description

## Link to Appium logs
will provided as needed

## Code To Reproduce Issue [ Good To Have ]
Each of the following is not working:
- `ActionChains(self.__driver).send_keys_to_element(element, text).perform()`
- `ActionChains(self.__driver).move_to_element(element).send_keys(text).perform()`
- `ActionChains(self.__driver).send_keys(text).perform()`, I called this after performing `element.click()`
- `element.send_keys(text)`
- `self.__driver.switch_to.active_element.send_keys(text)`

**Comment by bwomsm1** [comment]
Problem solved by calling the WebElement `send_value` api:
`element.set_value(text)`

Not sure its a proper solution but it did solve my issue.

**Comment by bwomsm1** [comment]
Problem solved by calling to `set_value()` WebElement api:
`element.set_value(text)`

**Comment by bwomsm1** [comment]
@mykola-mokhnach @KazuCocoa I am reopening the ticket since I am still having the issue also with `set_value` api.
This issue is quite blocking our project since we cannot pass login screen and continue the testing. 

Can you please take a look? I can share an ipa with you guys.

**Comment by KazuCocoa** [comment]
I quickly checked with https://github.com/appium/ios-uicatalog , but a secure text field in TextFields section worked as below.

```
> e = @driver.find_element :class, 'XCUIElementTypeSecureTextField'
> e.send_keys 'a'
```
My env was 1.22 rc and iOS 15 beta.

I saw a case in probably appium forum or issues that if the keyboard was not English, sending texts to secure text field does not work. I also saw a case sometimes password works only `self.__driver.switch_to.active_element.send_keys(text)` (so cannot get the secure text field element).

I haven't tried an `ionic` app, so potentially it depends on their implementation for the secure field.

You also could see iOS system log with `showiOSLog: true` capability. If the root cause is in iOS system layer, nothing can do in Appium side.
